src/common/components/clusterWizard/clusterDetailsValidation.ts

```diff
@@ -5,7 +5,7 @@ import { getDefaultCpuArchitecture, OpenshiftVersionOptionType } from '../../typ
 import { TangServer } from '../clusterConfiguration/DiskEncryptionFields/DiskEncryptionValues';
 import { FeatureSupportLevelData } from '../featureSupportLevels';
 import {
-  dnsNameValidationSchema,
+  baseDomainValidationSchema,
```

Comment:
The function `getClusterDetailsValidationSchema` is being used in CIM (`ClusterDeploymentDetailsStep`), so this change would affect it too.

Reply:
Ok, I'll take a look